server/credentials.go

```diff
@@ -85,17 +85,21 @@ func BuildARN(role string, defaultAccount string, accountAliases *map[string]str
 }
 
 func (s *directSessionTokenService) AssumeRole(user *User, role string, enableLDAPRoles bool) (*sts.Credentials, error) {
-	var arn string = BuildARN(role, s.iamAccount, s.accountAliases)
+	var arn = BuildARN(role, s.iamAccount, s.accountAliases)
```

Comment:
It's still a `string` though, right?

Reply:
Yes, but the type is not necessary and my tools are telling me it's bad golang form.

Reply:
ack